README.md

```diff
@@ -1,2 +1,41 @@
-# bubble_sort
-The bubble sort algorithm made in Ruby 
+# Bubble_sort
+
+> This is the first project in the Ruby module 
+
+## PREREQUISITE
+
+- Install at least one code editor 
+- Install Ruby to exectute the project
+
+## GETTING STARTED
+
+- clone the repository `$ git clone https://github.com/pacyL2K19/bubble_sort.git`
+- Run the command ruby bubble_sort.rb 
```

Comment:
- [ ] You don't have a bubble_sort.rb file to execute. Instead you have 'bubble_sort_by' and 'bubble_sort_secondary' files. Please kindly update your instructions to execute the file properly.